### DISCUSSION TITLE
part-ii/2021-1/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#155] [Question comments]
Part II 2021 Algebraic Geometry: Paper 2, Section II, I 

https://questions.tripos.org/part-ii/2021-1/

**Comment by callumhobbis** [comment]
I can show $GL(n,k)$ is open - it's just the complement of the zero locus of $\det$. But how can we talk about smoothness and irreducibility if it's the _complement_ of a variety, and not a variety? Then later, we talk about a "closed subvariety" - all varieties are closed by definition, right?

**Reply by artie2000** [reply]
There were slightly different definitions last year. Using this year's stuff, first embed GL as a closed affine variety, then do the irreducibility and smoothness.

**Comment by Kiolt** [comment]
<details>
<summary>Embedding as a variety.</summary>

Any distinguished open subset $\mathbb{V}(f)^c$ of $\mathbb{A}^n$ can be embedded as the variety $\mathbb{V}(1-x_{n+1}f)$ in $\mathbb{A}^{n+1}$.

</details>

<details>
<summary>Affine morphisms</summary>

Nullstellensatz.

</details>

<details>
<summary>Projective morphisms</summary>

Take a representative $(F:G)$ for the morphism, then the fibre of $(a:b)$ contains an open subset of the irreducible hypersurface $V_{(a:b)}=\mathbb{V}(aG-bF)$. But fibres are closed, so $V_{(a:b)}$ is exactly the fibre. But any two such hypersurfaces intersect.

</details>
